zgw/notificaties-api/src/main/kotlin/com/ritense/notificatiesapi/autoconfigure/NotificatiesApiLiquibaseAutoConfiguration.kt

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright 2015-2023 Ritense BV, the Netherlands.
+ *
+ * Licensed under EUPL, Version 1.2 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://joinup.ec.europa.eu/collection/eupl/eupl-text-eupl-12
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" basis,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.ritense.notificatiesapi.autoconfigure
+
+import com.ritense.valtimo.contract.config.LiquibaseMasterChangeLogLocation
+import javax.sql.DataSource
+import org.springframework.boot.autoconfigure.condition.ConditionalOnClass
+import org.springframework.context.annotation.Bean
+import org.springframework.context.annotation.Configuration
+import org.springframework.core.Ordered
+import org.springframework.core.annotation.Order
+
+@Configuration
+@ConditionalOnClass(DataSource::class)
+class NotificatiesApiLiquibaseAutoConfiguration {
+
+    @Order(Ordered.HIGHEST_PRECEDENCE + 12)
```

Comment:
Could you change this to `+ 25`?
To ensure that this liquibase script is executed after all other liquibase scripts inside Valtimo

Reply:
Adressed.

Is this specified somewhere what order we want what liquibase changesets to run? I assume the reason here is so that valtimo/gzac core can always ensure they can run their changesets before any additional features are deployed.
Maybe worth noting this somewhere, so that people who want to contribute know that precedence <25 is reserved.